src/components/Collapsible/tests/Collapsible.test.tsx

```diff
@@ -17,22 +17,6 @@ describe('<Collapsible />', () => {
 
     const hidden = collapsible.find(ariaHiddenSelector);
     expect(hidden.exists()).toBe(true);
-    expect(collapsible.contains('content')).toBe(false);
-  });
-
-  it('does not render its children when going from open to closed', () => {
```

Comment:
Unfortunately, this is an important test. Some consumer implementations rely on this - I'm sure you're going to see some tests failing in web

Reply:
Is it necessary to not render or tests will now fail?

Reply:
Tests will probably fail (unless the code has been refactored 🤞 ) - it might be easier to release a rc and run CI with that branch. I don't *think* UI will break - but it would be worthwhile checking the failed tests.

Reply:
I'd like to remind y'all about https://buildkite.com/shopify/polaris-react-system-integration-test which runs a pass of the web CI on arbitary polaris branches.

It runs by default on our master branch, but it'd be trivial to trigger a manual build that points at this branch.

No faffing with RCs needed.